lib/Doctrine/ORM/Internal/Hydration/ScalarColumnHydrator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Internal\Hydration;
+
+use Doctrine\DBAL\Driver\Exception;
+use Doctrine\ORM\Exception\MultipleSelectorsFoundException;
+
+use function count;
+use function current;
+use function is_array;
+
+/**
+ * Hydrator that produces one-dimensional array.
+ */
+final class ScalarColumnHydrator extends AbstractHydrator
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws MultipleSelectorsFoundException
+     * @throws Exception
+     */
+    protected function hydrateAllData(): array
+    {
+        if (count($this->resultSetMapping()->fieldMappings) > 1) {
+            throw MultipleSelectorsFoundException::create($this->resultSetMapping()->fieldMappings);
+        }
+
+        $result = [];
+
+        while ($row = $this->statement()->fetchOne()) {
+            if (is_array($row)) {
+                $result[] = current($row);
+                continue;
```

Comment:
Codecov sees uncovered code here.

Reply:
with new implementation of `fetchOne()`, these lines not required, removed it.